tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml

```diff
@@ -0,0 +1,179 @@
+                <Setter Property="Background" Value="Transparent" />
+                <Setter Property="Content" Value="&#xE712;" />
+                <Setter Property="FontFamily" Value="{StaticResource SymbolThemeFontFamily}" />
+                <Setter Property="HorizontalAlignment" Value="Right" />
+                <Setter Property="HorizontalAlignment" Value="Right" />
+                <Setter Property="BorderThickness" Value="0" />
+                <Setter Property="Width" Value="50" />
+                <Setter Property="Height" Value="36" />
+                <Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+            </Style>
+        </ResourceDictionary>
+    </commonviews:ToolPage.Resources>
+    <Grid>
+        <Grid.Resources>
+            <!--  Three Dot Button template  -->
+            <DataTemplate x:Key="ThreeDotsButton">
+                <Button x:Uid="MoreOptionsButton" Style="{StaticResource HorizontalThreeDotsStyle}">
+                    <Button.Flyout>
+                        <MenuFlyout>
+                            <MenuFlyoutItem Text="Open in File Explorer" />
+                            <MenuFlyoutItem Text="Open in Command Line" />
+                            <MenuFlyoutItem Text="Move Repository" />
+                            <MenuFlyoutItem Text="Add to Config file" />
+                            <MenuFlyoutItem Text="Run to config file" />
+                            <MenuFlyoutItem Text="Remove from this list" />
+                            <MenuFlyoutItem Text="Delete Repository" />
+                        </MenuFlyout>
+                    </Button.Flyout>
+                </Button>
+            </DataTemplate>
+        </Grid.Resources>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <Grid
+            x:Name="TitleGrid"
+            Grid.Row="0"
+            MaxWidth="{ThemeResource MaxPageContentWidth}"
+            Margin="{ThemeResource HeaderMargin}"
+            ColumnSpacing="5">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition Width="auto" />
+                <ColumnDefinition Width="auto" />
+            </Grid.ColumnDefinitions>
+            <TextBlock
+                x:Name="EnvironmentsHeader"
+                Grid.Column="0"
+                HorizontalAlignment="Left"
+                Style="{ThemeResource SubtitleTextBlockStyle}"
+                Text="Repositories" />
+            <Button Grid.Column="1" Content="Add repository" />
+            <Button
+                Grid.Column="2"
+                Margin="0,0,50,0"
+                Content="Clone repository" />
+        </Grid>
+        <Grid
+            Grid.Row="1"
+            Margin="{ThemeResource HeaderMargin}"
```

Comment:
Is this not supposed to conform to MaxPageContentWidth? (Same question for grid in row 2)